<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>482</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>4096</width>
    <height>4096</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,4,2">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinAndMaxSize</enum>
       </property>
       <item>
        <widget class="QGroupBox" name="groupOperate">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>400</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>4096</height>
          </size>
         </property>
         <property name="title">
          <string>操作和常数</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>卡组容量：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="deckNum">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>25</height>
             </size>
            </property>
            <property name="decimals">
             <number>0</number>
            </property>
            <property name="minimum">
             <double>0.000000000000000</double>
            </property>
            <property name="maximum">
             <double>60.000000000000000</double>
            </property>
            <property name="value">
             <double>40.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>手坑：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="shoukengNum">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>25</height>
             </size>
            </property>
            <property name="decimals">
             <number>0</number>
            </property>
            <property name="minimum">
             <double>0.000000000000000</double>
            </property>
            <property name="maximum">
             <double>60.000000000000000</double>
            </property>
            <property name="stepType">
             <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
            </property>
            <property name="value">
             <double>0.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>后场坑：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="houkengNum">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>25</height>
             </size>
            </property>
            <property name="decimals">
             <number>0</number>
            </property>
            <property name="minimum">
             <double>0.000000000000000</double>
            </property>
            <property name="maximum">
             <double>60.000000000000000</double>
            </property>
            <property name="value">
             <double>0.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="NumLable">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupInput">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>400</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>4096</width>
           <height>4096</height>
          </size>
         </property>
         <property name="title">
          <string>输入</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,2">
          <property name="sizeConstraint">
           <enum>QLayout::SetMaximumSize</enum>
          </property>
          <item>
           <widget class="QTextEdit" name="textEdit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="html">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Microsoft YaHei UI';&quot;&gt;说明：&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Microsoft YaHei UI';&quot;&gt;所有展开组件分组参与计算，比如：&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Microsoft YaHei UI';&quot;&gt;投入了一个单卡展开的组件，一共三张，&lt;/span&gt;&lt;span style=&quot; font-family:'Microsoft YaHei UI'; font-weight:696;&quot;&gt;则N卡展开为1、组件1为3、组件2为0、组件3为0；&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Microsoft YaHei UI';&quot;&gt;投入了一个双卡展开的组件，其中一个是三张，另一个是一张，&lt;/span&gt;&lt;span style=&quot; font-family:'Microsoft YaHei UI'; font-weight:696;&quot;&gt;则N卡展开为2、组件1为3、组件2为1、组件3为0；&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Microsoft YaHei UI'; font-weight:696;&quot;&gt;则N卡展开为0时，这一列的数据不会被启用。&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Microsoft YaHei UI';&quot;&gt;行1-10是指的你可以输入的1-10组独立的展开组合。&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Microsoft YaHei UI';&quot;&gt;demo暂时不支持更多组数，以及非独立展开组合的计算。&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="inPutWidget">
            <property name="columnCount">
             <number>10</number>
            </property>
            <row>
             <property name="text">
              <string>N卡展开</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>组件1投入数</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>组件2投入数</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>组件3投入数</string>
             </property>
            </row>
            <column/>
            <column/>
            <column/>
            <column/>
            <column/>
            <column/>
            <column/>
            <column/>
            <column/>
            <column/>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupOutput">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>400</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>4096</width>
           <height>4096</height>
          </size>
         </property>
         <property name="title">
          <string>上手率-计算结果</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="sizeConstraint">
           <enum>QLayout::SetMaximumSize</enum>
          </property>
          <item>
           <widget class="QTableWidget" name="outPutWidget">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sortingEnabled">
             <bool>false</bool>
            </property>
            <attribute name="horizontalHeaderVisible">
             <bool>true</bool>
            </attribute>
            <attribute name="horizontalHeaderCascadingSectionResizes">
             <bool>true</bool>
            </attribute>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>43</number>
            </attribute>
            <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderCascadingSectionResizes">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderShowSortIndicator" stdset="0">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderStretchLastSection">
             <bool>false</bool>
            </attribute>
            <row>
             <property name="text">
              <string>手坑数</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>后场坑数</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>单卡展开件数</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>双卡展开组数</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>三卡展开组数</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>无坑展开率</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>吃一坑展开率</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>吃二坑展开率</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>废件数</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string/>
             </property>
            </row>
            <column>
             <property name="text">
              <string>先手</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>抽一</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>抽二</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>926</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionimport"/>
    <addaction name="actionexport"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionimport">
   <property name="text">
    <string>导入记录</string>
   </property>
  </action>
  <action name="actionexport">
   <property name="text">
    <string>导出记录</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
